tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,33 @@
 """
 
 
+test_include_paths_file = """
+#include "foo.h"
+#include "bar.h"
+
+int main() {
+    foo();
+    bar();
+    return 0;
+}
+"""
+
+
+def get_testdirectory_name(test_file):
+    return os.path.join(os.path.dirname(__file__), 'test_files', test_file)
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,
+                                                    test_include_paths_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearValidIncludePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(test_include_paths_file,),
+    invalid_files=(),
+    settings={'include_paths': get_testdirectory_name('headers1')+',' +
+              get_testdirectory_name('headers2')},
+    tempfile_kwargs={'suffix': '.cpp'}
+    )
```

Comment:
to keep the same style, I would move the closing bracket `)` to the previous line^^